internal/config/config.go

```diff
@@ -56,6 +56,10 @@ func (c *Config) IsAuthenticated() error {
 		return errors.New("both --service-token and --service-token-id are required for service token authentication")
 	}
 
+	if c.ServiceTokenIsSwapped() {
+		return errors.New("the --service-token and --service-token-id values are swapped")
```

Comment:
Wonder if it's nicer to say to the user "it looks like you swapped..."? No super strong opinions there though.

Reply:
How does `the --service-token and --service-token-id values appear to be swapped` sound to you?

Reply:
Updated the wording of the message to your example so it should look more like this now:

```
Error: not authenticated yet. Please run 'pscale auth login'
Error: it looks like you swapped the --service-token and --service-token-id values
```